packages/components/Callout/macos/CalloutManager.swift

```diff
@@ -1,4 +1,6 @@
+import AppKit
 import Foundation
```

Comment:
Should we still have the Foundation import?

Reply:
They are separate module so good to import what we need